src/metadata/routes.clj

```diff
@@ -18,27 +17,27 @@
 
 (defapi app
   {:exceptions cx/exception-handlers}
-  (swagger-ui config/docs-uri
-    :validator-url nil)
-  (swagger-docs
-    {:info {:title "Discovery Environment Metadata API"
-            :description "Documentation for the Discovery Environment Metadata REST API"
-            :version "2.0.0"}
-     :tags [{:name "service-info", :description "Service Information"}
-            {:name "avus", :description "Attribute/Value/Unit Management"}
-            {:name "permanent-id-requests", :description "Permanent ID Request Management"}
-            {:name "admin-permanent-id-requests", :description "Admin Permanent ID Request Mgmt"}
-            {:name "data-comments", :description "Comments on Data Items"}
-            {:name "app-comments", :description "Comments on Apps"}
-            {:name "favorites", :description "Favorite Resources"}
-            {:name "ontologies", :description "Ontology Hierarchies"}
-            {:name "admin-ontologies", :description "Ontology Hierarchy Administration"}
-            {:name "admin-data-comments", :description "Admin Data Item Comment Management"}
-            {:name "admin-app-comments", :description "Admin App Comment Management"}
-            {:name "tags", :description "Tags Management"}
-            {:name "template-info", :description "Template Information"}
-            {:name "template-administration", :description "Template Administration"}]})
-  (middlewares
+  (swagger-routes
+    {:ui      config/docs-uri
+     :options {:ui {:validatorUrl nil}}
+     :data    {:info {:title "Discovery Environment Metadata API"
+                      :description "Documentation for the Discovery Environment Metadata REST API"
+                      :version "2.8.0"}
```

Comment:
I have no idea if we have a plan for versioning the API stuff like this; before we just always kept it at 2.0. It seems like if we're updating it we should update it when the API changes, to match the version in project.clj, but that also seems like potentialy a lot of  work. Anyway, probably fine for now.